### PR TITLE
Suggest using 64bit JVM for Artery in docs

### PR DESCRIPTION
**Pull request by andrievskya**
https://github.com/akka/akka/issues/22303

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @andrievskya,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by andrievskya** [comment]
Signed the Typesafe CLA

File: akka-docs/rst/java/remoting-artery.rst [hunk 1]
```diff
@@ -87,6 +87,9 @@ As you can see in the example above there are four things you need to add to get
 The example above only illustrates the bare minimum of properties you have to add to enable remoting.
 All settings are described in :ref:`remote-configuration-artery-java`.
 
+.. warning::
+  Aeron requires 64bit JVM for reliable work.
```
Comment (johanandren) [inline akka-docs/rst/java/remoting-artery.rst]: "to work reliably"

**Review by johanandren — COMMENTED** [review]
Thanks. Did you try it on 32bit and have problems? (It never even surfaced to me that people might be using 32bit JVMs)

**Comment by johanandren** [comment]
NVM question about 32bit, saw ticket now.

File: akka-docs/rst/java/remoting-artery.rst (hunk 1 above)
Comment (drewhk) [inline akka-docs/rst/java/remoting-artery.rst]: +1

**Review by drewhk — APPROVED** [review]
Small changes to wording required

File: akka-docs/rst/java/remoting-artery.rst (hunk 1 above)
Comment (andrievskya) [inline akka-docs/rst/java/remoting-artery.rst]: yes, that sounds better

Comment (patriknw) [inline akka-docs/rst/java/remoting-artery.rst]: same thing in `scala/remoting-artery.rst`

not sure that this deserves `warning` (red box). I think `note` is enough. 64 bit is default in most environments

**Review by patriknw — CHANGES_REQUESTED** [review]

**Comment by patriknw** [comment]
@andrievskya Thanks for adding this. We have different pages for java and scala. Can you please add the same thing to `akka-docs/rst/scala/remoting-artery.rst`. Then we can merge this.

**Comment by andrievskya** [comment]
Added the same text to file you mentioned. Do I need to create another pull request, or something, so you can integrate changes into where you need, and I can close the issue?

**Comment by johanandren** [comment]
Just commit and push to your github repo and they will appear in this PR.

**Comment by andrievskya** [comment]
Added the same text to file you mentioned. Do I need to create another pull request, or something, so you can integrate changes into where you need, and I can close the issue?

**Comment by patriknw** [comment]
@andrievskya would you like to complete this, or shall I do the copy?

**Comment by andrievskya** [comment]
Do you mean closing pull request? or there are some meaningfull work left for me to do?

**Comment by patriknw** [comment]
The remaining, that we are waiting for:

> We have different pages for java and scala. Can you please add the same thing to akka-docs/rst/scala/remoting-artery.rst.

**Comment by andrievskya** [comment]
These changes are in commit 669543c9ee0699c149755a1f67b45603bb58862d in fork andrievskyaa/akka
(https://github.com/andrievskya/akka/commit/669543c9ee0699c149755a1f67b45603bb58862d) what do I need to do to put these changes where you need them?

**Comment by patriknw** [comment]
Ok, push to the branch you used for this PR and they will show up here. Thanks. If trouble, just create a new PR.

**Comment by andrievskya** [comment]
Created new pull request #22553 that (I hope) contains all the changes discussed here

**Pull request closed without merging**